metagraph-katana-plugin/metagraph_katana/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ._version import get_versions
+
+__version__ = get_versions()["version"]
```

Comment:
We have our own version generation code. You can get your version from `katana.__version__`. Eventually I want to have separate versions built into each library, but for now we can just all use the same version (which actually comes from the version built into libsupport).